src/assets/TokenRatesController.test.ts

```diff
@@ -271,4 +278,92 @@ describe('TokenRatesController', () => {
     // FIXME: This is now being called twice
     expect(updateExchangeRatesStub.callCount).toStrictEqual(2);
   });
+
+  it('should update exchange rates when native currency is not supported by coingecko', async () => {
+    nock(COINGECKO_HOST)
+      .get(COINGECKO_ASSETS_PATH)
+      .reply(200, [
+        {
+          id: 'binance-smart-chain',
+          chain_identifier: 56,
+          name: 'Binance Smart Chain',
+          shortname: 'BSC',
+        },
+        {
+          id: 'ethereum',
+          chain_identifier: 1,
+          name: 'Ethereum',
+          shortname: '',
+        },
+        {
+          id: 'polygon-pos-network',
+          chain_identifier: 137,
+          name: 'Polygon',
+          shortname: 'MATIC',
+        },
+      ])
+      .get(`${COINGECKO_MATIC_PATH}`)
+      .query({ contract_addresses: '0x02,0x03', vs_currencies: 'eth' })
+      .reply(200, {
+        '0x02': {
+          eth: 0.001, // token value in terms of ETH
+        },
+        '0x03': {
+          eth: 0.002,
+        },
+      })
+      .persist();
+
+    nock('https://min-api.cryptocompare.com')
+      .get('/data/price?fsym=ETH&tsyms=MATIC')
+      .reply(200, { MATIC: 0.5 }) // .5 eth to 1 matic
+      .persist();
+
+    const expectedExchangeRates = {
+      '0x02': 0.0005, // token value in terms of matic = (token value in eth) / (matic value in eth) = .001 / 2
```

Comment:
Nit: We're now multiplying by the ETH => MATIC price rather than dividing by the MATIC => ETH price
```suggestion
      '0x02': 0.0005, // token value in terms of matic = (token value in eth) * (eth value in matic) = .001 / 2
```